paddlets/datasets/tsdataset.py

```diff
@@ -255,7 +255,7 @@ def astype(self, dtype: Union[np.dtype, type, Dict[str, Union[np.dtype, type]]])
         self._data = self._data.astype(dtype)
         return self
 
-    def to_dataframe(self, copy: bool=True) -> pd.DataFrame:
+    def to_dataframe(self, copy: Optional[bool]=True) -> pd.DataFrame:
```

Comment:
Line258 copy should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional